src/main/java/seedu/address/model/person/Name.java

```diff
@@ -38,6 +38,30 @@ public static boolean isValidName(String test) {
         return test.matches(VALIDATION_REGEX);
     }
 
+    private static String formatName(String name) {
+        String[] words = name.toLowerCase().split("\\s+");
+        StringBuilder formattedWords = new StringBuilder();
+
+        for (String word : words) {
+            if (!word.isEmpty()) {
+                formattedWords.append(capitalizeWord(word)).append(" ");
+            }
+        }
```

Comment:
Looks better and easier to understand

Reply:
You wna abstract the loop also? To look neater

Reply:
Thank you for the suggestion! Ive abstracted the initial format name into 3 different methods that better reflects the process of formatting the name.